src/functions.php

```diff
@@ -15,4 +15,5 @@ function lambda(callable $handler): void
 {
     $lambdaRuntime = Bref\Runtime\LambdaRuntime::fromEnvironmentVariable();
     $lambdaRuntime->processNextEvent($handler);
+    exit(0);
```

Comment:
Remember to add some clear documentation in the deprecation notice about this not being compatible with multiple invocations